resources/js/icons.tsx

```diff
@@ -70,6 +70,7 @@ import { ReactComponent as Metamask } from "@icons/metamask.svg";
 import { ReactComponent as Minus } from "@icons/minus.svg";
 import { ReactComponent as DiamondOpacity } from "@icons/opacity/diamond.svg";
 import { ReactComponent as ImageOpacity } from "@icons/opacity/image.svg";
+import { ReactComponent as NewspaperOpacity } from "@icons/opacity/newspaper.svg";
```

Comment:
why is this inside the `opacity` folder?

Reply:
@ItsANameToo really because rest of the icons on that tab are there, didnt really checked if it has opacity